client/thin-replica-client/include/client/thin-replica-client/trs_connection.hpp

```diff
@@ -147,6 +150,21 @@ class TrsConnection {
   virtual Result readStateHash(const com::vmware::concord::thin_replica::ReadStateHashRequest& request,
                                com::vmware::concord::thin_replica::Hash* hash);
 
+  // Open a state snapshot subscription stream (connection has to be established before).
+  // A state snapshot subscription stream will be open after
+  // openStateSnapshotStream returns, if and only if openStateSnapshotStream returns
+  // Result::kSuccess; a stream will not be open in the failure and timeout cases.
```

Comment:
I think the word "subscription" is misplaced here. From my point of view, you subscribe to all updates that exist now and **in the future**. For state snapshots, the moment you call the API, the updates already exists and are ready to be consumed. Meaning, there are no updates that you're waiting on to be produced.
Therefore, I would just call it a stream, no subscription (same comment below).

Reply:
Updated.